Sources/OpenAI/Public/Models/ChatQuery.swift

```diff
@@ -7,10 +7,104 @@
 
 import Foundation
 
+
+
+// See more https://platform.openai.com/docs/guides/text-generation/json-mode
+public struct ResponseFormat: Codable, Equatable {
```

Comment:
Would be nice to bring it out to the separate file. As I understand it relates only to chats, so let's perhaps name it smth like `ChatQueryResponseFormat` or `ChatQueryJSONResponse` 🤔

Reply:
what about `ChatQuery.ResponseFormat`?

Reply:
Alright, but let's make it as an extension for more clarity and code being more swiftly 🙌

Reply:
I'd split the chat part into separate files too, maybe into a subfolder smth like 
![CleanShot 2023-11-14 at 13 56 38@2x](https://github.com/MacPaw/OpenAI/assets/16429579/ba1ee676-083b-405a-9807-2fe3a4cf1112)

Reply:
Perfect! ❤️